kmcd-prelude.md

```diff
@@ -233,11 +234,20 @@ module KMCD-GEN
  // -----------------------------------
     rule <k> GenStep => GenStepLoad ~> GenStepReplace ... </k>
          <random> BS => tail(BS) </random>
+         <used-random> _ => Int2Bytes(head(BS), 1, BE) </used-random>
          <generator-next> N </generator-next>
          <generator-current> _ => head(BS) modInt N </generator-current>
          <generator-remainder> GSS => .GenStep </generator-remainder>
          <violation> false </violation>
       requires lengthBytes(BS) >Int 0
+       andBool N >Int 0
+
+    rule <k> GenStep => . ... </k>
+         <generator-next> N </generator-next>
+         <random> BS => tail(BS) </random>
+         <used-random> BS' => BS' +Bytes Int2Bytes(head(BS), 1, BE) </used-random>
+      requires lengthBytes(BS) >Int 0
+       andBool notBool N >Int 0
```

Comment:
I'd move `<generator-next> N </generator-next>` two lines below for consistency with other similar rules.

Reply:
Done.